specification/components/stu3/schemas/endpoints/a025-retrieve-advice-and-guidance-conversation.yaml

```diff
@@ -35,6 +42,16 @@ description: |
   ## Important note
 
   The conversation returned includes links to any attachments associated with the conversation.
+  This can be done via endpoint [[HYPERLINK_A006]]. 
+  Note, dependent on how attachments were added they may be linked to the Advice and Guidance summary instead of the conversation. 
+  These may be retrieved from [[HYPERLINK_A024]].
+
+  If the converted advice and guidance request is referred to a service outside of your organisation, 
+  the subsequent referral information may not be accessible due to existing data-access rules. 
+  However, the advice and guidance summary ([[HYPERLINK_A024]]), attachments ([[HYPERLINK_A006]]), 
+  advice and guidance conversation ([[HYPERLINK_A025]]) and 
+  clinical information ([[HYPERLINK_A043]]) which took place before such a conversion still forms part of the clinical information and can still be retrieved.
+  The conversation also gets included in the form of a PDF file returned by [[HYPERLINK_A007]]
 
   ## Related endpoints
 
```

Comment:
Should the sandbox work for A025? Also On ERSSUP-63516 it shows GET/CommunicationRequest/{ubrn} but the 'Try this API' link doesn't include the /{ubrn}

Reply:
Yes it should work. The actual endpoint url is "/STU3/Communication". I think it was copied from A024 endpoint to here https://nhsd-jira.digital.nhs.uk/browse/ERSSUP-63516.  I will raise this to BA

Reply:
BA updated the Story